docs/ios/xcsync.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Xcode Sync (xcsync) for .NET MAUI
+description: [Xcsync is designed to provide .NET developers with improved support for editing Apple specific files]
+author: haritha-mohan, mcumming
```

Comment:
You should use `ms.author` for your MS aliases, and then `author` for your GitHub aliases.